src/game.py

```diff
@@ -1,48 +1,76 @@
 import math
-from pawn import Pawn, translate_x, translate_y
+from pawn import Orientation, Pawn, translate_x, translate_y
 from exception import QuoridorException, OutOfBoardException, UnknownActionException
 from functools import reduce
 from action import Action
 import console
 from board import *
 
+from enum import IntEnum
+
+
+class State(IntEnum):
+    RUNNING = 0,
+    VICTORY = 1,
+    QUIT = 2,
+
 
 def init_game():
     center = math.floor(BASE_LINE_SIZE / 2)
-    return Pawn(0, center)
+    return [Pawn(0, center, Orientation.WEST), Pawn(BASE_LINE_SIZE - 1, center, Orientation.EAST)]
 
 
-def progress(pawn):
-    new_pawn = pawn
+def progress(pawns):
+    new_pawns = pawns
```

Comment:
Be aware: you didn't copied the value of your pawns, you just stored the reference of your pawns.
So, if you change new_pawns later, pawns will be changed too